javascript/javascript-core/intro-js/why-learn-javascript.md

```diff
@@ -59,3 +63,15 @@ Why was **JavaScript** originally developed?
 - To enhance Java
 - To replace HTML
  
+---
+
+## Footnotes
+
+[1: JavaScript Rules The World]
+JavaScript has been the most widely used programming language in the world for the good part of the last decade.
+
+[!Statista_Popular_Languages](https://img.enkipro.com/21314c0f349267942234c15e076c4f85.png)
+> source: [Statista](https://www.statista.com/statistics/793628/worldwide-developer-survey-most-used-languages/)
```

Comment:
hmm, maybe we need a way to add reference the original source of images in the app

Reply:
Like tag the photo directly, instead of having to add the source below?